### PR TITLE
Connectionstring upgrade

### PR DESCRIPTION
**Pull request by dbelcham**
This is related to the changes in https://github.com/Particular/NServiceBus/pull/4891

### CODE REVIEW

File: Snippets/Core/Core_7/UpgradeGuides/6to7/ConnectionStrings/ConfigurationChanges.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    using NServiceBus;
+
+    public class ConfigurationChanges
+    {
+
+        void ProvideConfiguration(EndpointConfiguration endpointConfiguration)
+        {
+            #region 6to7ConnectionStrings
+            var connectionString = ConfigurationManager.ConnectionStrings["theConnectionName"].ConnectionString;
+            transport.ConnectionString(connectionString);
```
Comment (SimonCropp) [inline Snippets/Core/Core_7/UpgradeGuides/6to7/ConnectionStrings/ConfigurationChanges.cs]: how is this meant to compile?

File: transports/sql/connection-settings_connection-string_sqltransport_[4,).partial.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+### Via the configuration API
```
Comment (SimonCropp) [inline transports/sql/connection-settings_connection-string_sqltransport_[4,).partial.md]: u cant use a core snippet in a doco page that targets the sql transport component

**Comment by SimonCropp** [comment]
@Particular/nservicebus-maintainers @Particular/sqlserver-transport-maintainers can you review this

File: nservicebus/upgrades/6to7/index.md [hunk 3]
```diff
@@ -211,4 +211,13 @@ WARNING: The `{AppDomain.CurrentDomain.BaseDirectory}\License\License.xml` path
 
 When running on the full .NET Framework, endpoints will [continue to search the registry locations](/nservicebus/licensing/) for a suitable license.
 
-When running on the .NET Core platform, endpoints **will not search the registry**, even when running on Windows.
\ No newline at end of file
+When running on the .NET Core platform, endpoints **will not search the registry**, even when running on Windows.
+
+
+## Connection Strings
+
```
Comment (andreasohlund) [inline nservicebus/upgrades/6to7/index.md]: Should also mention that the implicit "NServiceBus/Transport" connection string should be avoided? (Or is the runtime WARN log enough?)

Comment (SimonCropp) [reply nservicebus/upgrades/6to7/index.md]: I think mention it is a good idea

File: nservicebus/upgrades/6to7/index.md [hunk 4]
```diff
@@ -220,4 +220,7 @@ When running on the .NET Core platform, endpoints **will not search the registry
 ### Named connection strings
 Configuring a transport's connection using `.ConnectionStringName(name)` has been removed. To continue to retrieve the connection string by the named value in the configuration, first retrieve the connection string and then pass it to the `.ConnectionString(value)` configuration.
 
-snippet: 6to7ConnectionStrings
\ No newline at end of file
+### Implicit "NServiceBus/Transport" connection string use
+NServiceBus will no longer automatically find and use a connection string that is named `NServiceBus/Transport` by default. The connection string value must be explicitly configured using `.ConnectionString(value)`.
```
Comment (andreasohlund) [inline nservicebus/upgrades/6to7/index.md]: I thought we did read it in net452 but logged a WARN?

**Comment by dbelcham** [comment]
@Particular/docs-maintainers I'm confused as to why the builds won't pass. The problems seem to be related to things that were never touched in this PR, yet the failures persist. Any ideas?

**Comment by SimonCropp** [comment]
U need to rebase

**Comment by weralabaj** [comment]
rebased

**Pull request merged**